yolort/models/common.py

```diff
@@ -170,6 +170,15 @@ def focus_transform(x: Tensor) -> Tensor:
     return y
 
 
+def space_to_depth(x: Tensor) -> Tensor:
```

Comment:
We don't use this function yet, but we'll put it here for now and see how to call this part later, or do some experiments with it.